app/views/monthlyreturns/DateConfirmNilPaymentsView.scala.html

```diff
@@ -0,0 +1,93 @@
+@*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import views.html.components.*
+@import uk.gov.hmrc.hmrcfrontend.views.html.components.implicits._
+
+@this(
+    layout: templates.Layout,
+    h1: H1,
+    paragraph: Paragraph,
+    formHelper: FormWithCSRF,
+    govukErrorSummary: GovukErrorSummary,
+    govukDateInput: GovukDateInput,
+    govukButton: GovukButton,
+    govukWarningText: GovukWarningText,
+    govukDetails: GovukDetails
+)
+
+@(form: Form[_], mode: Mode)(implicit request: Request[_], messages: Messages)
+
+@layout(pageTitle = title(form, messages("monthlyreturns.dateConfirmNilPayments.title"))) {
+
+    @formHelper(action = controllers.monthlyreturns.routes.DateConfirmNilPaymentsController.onSubmit(mode), Symbol("autoComplete") -> "off") {
+
+        @if(form.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummaryViewModel(form, errorLinkOverrides = Map("value" -> "value.day")))
+        }
+
+        @h1("monthlyreturns.dateConfirmNilPayments.title")
+        @paragraph("monthlyreturns.dateConfirmNilPayments.paragraph")
+
+
+        @govukDateInput(
+            WithMonthYearViewModel(
+                field  = form("value"),
+                legend = LegendViewModel(
+                    HtmlContent(
+                        Seq(
+                            s"<dt><b>${messages("monthlyreturns.dateConfirmNilPayments.heading.h2")}</b></dt>",
```

Comment:
Suggest we use an H3 tage for the legend content:

                    HtmlContent(
                        Seq(
                            s"<h3>${messages("monthlyreturns.dateConfirmNilPayments.heading.h2")}</h3>"
                        ).mkString
                    )

Reply:
As per earlier suggestion by Waigwa, changed it to h2.